lib/stdlib/src/supervisor.erl

```diff
@@ -216,102 +289,103 @@ delete_child(Supervisor, Id) ->
 check_childspecs(ChildSpecs) when is_list(ChildSpecs) ->
     case check_startspec(ChildSpecs) of
-	{ok, _} -> ok;
-	Error -> {error, Error}
+        {ok, _} -> ok;
+        Error -> {error, Error}
     end;
-check_childspecs(X) -> {error, {badarg, X}}.
+check_childspecs(X) ->
+    {error, {badarg, X}}.
 
 %%%-----------------------------------------------------------------
 %%% Called by release_handler during upgrade
 -spec get_callback_module(Pid) -> Module when
-      Pid :: pid(),
-      Module :: atom().
+    Pid :: pid(),
+    Module :: atom().
 get_callback_module(Pid) ->
-    {status, _Pid, {module, _Mod},
-     [_PDict, _SysState, _Parent, _Dbg, Misc]} = sys:get_status(Pid),
+    {status, _Pid, {module, _Mod}, [_PDict, _SysState, _Parent, _Dbg, Misc]} = sys:get_status(Pid),
     case lists:keyfind(supervisor, 1, Misc) of
-	{supervisor, [{"Callback", Mod}]} ->
-	    Mod;
-	_ ->
-	    [_Header, _Data, {data, [{"State", State}]} | _] = Misc,
-	    State#state.module
+        {supervisor, [{"Callback", Mod}]} ->
+            Mod;
+        _ ->
+            [_Header, _Data, {data, [{"State", State}]} | _] = Misc,
+            State#state.module
     end.
 
 %%% ---------------------------------------------------
-%%% 
+%%%
 %%% Initialize the supervisor.
-%%% 
+%%%
 %%% ---------------------------------------------------
 
 -type init_sup_name() :: sup_name() | 'self'.
 
--type stop_rsn() :: {'shutdown', term()}
-                  | {'bad_return', {module(),'init', term()}}
-                  | {'bad_start_spec', term()}
-                  | {'start_spec', term()}
-                  | {'supervisor_data', term()}.
+-type stop_rsn() ::
+    {'shutdown', term()} |
+    {'bad_return', {module(), 'init', term()}} |
+    {'bad_start_spec', term()} |
+    {'start_spec', term()} |
+    {'supervisor_data', term()}.
 
 -spec init({init_sup_name(), module(), [term()]}) ->
-        {'ok', state()} | 'ignore' | {'stop', stop_rsn()}.
-
+    {'ok', state()} | 'ignore' | {'stop', stop_rsn()}.
 init({SupName, Mod, Args}) ->
     process_flag(trap_exit, true),
     case Mod:init(Args) of
-	{ok, {SupFlags, StartSpec}} ->
-	    case init_state(SupName, SupFlags, Mod, Args) of
-		{ok, State} when ?is_simple(State) ->
-		    init_dynamic(State, StartSpec);
-		{ok, State} ->
-		    init_children(State, StartSpec);
-		Error ->
-		    {stop, {supervisor_data, Error}}
-	    end;
-	ignore ->
-	    ignore;
-	Error ->
-	    {stop, {bad_return, {Mod, init, Error}}}
+        {ok, {SupFlags, StartSpec}} ->
+            case init_state(SupName, SupFlags, Mod, Args) of
+                {ok, State} when ?is_simple(State) ->
+                    init_dynamic(State, StartSpec);
+                {ok, State} ->
+                    init_children(State, StartSpec);
+                Error ->
+                    {stop, {supervisor_data, Error}}
+            end;
+        ignore ->
+            ignore;
+        Error ->
+            {stop, {bad_return, {Mod, init, Error}}}
```

Comment:
There are lot of such indenting changes. In general I think we should adhere to how the Emacs mode indents it in these cases as that is the most spread way at least in the OTP code base. How well supervisor adheres I do not know some of this code is really old.

Reply:
In general we want to avoid alignment that is not at 4 spaces, but we also have an open issue to discuss emacs indentation https://github.com/WhatsApp/erlfmt/issues/166